threedi_modelchecker/checks/other.py

```diff
@@ -10,15 +10,6 @@
 from .cross_section_definitions import cross_section_configuration
 from .geo_query import distance, length, transform
 
-# Use these to make checks only work on the first global settings entry:
-first_setting = (
-    Query(models.GlobalSetting.id)
-    .order_by(models.GlobalSetting.id)
-    .limit(1)
-    .scalar_subquery()
-)
-first_setting_filter = models.GlobalSetting.id == first_setting
-
```

Comment:
No longer used because `ModelSettings` should only have one row

Reply:
OK, is there a test for these tables that must contain only one row?

Reply:
Yes, and based on your remarks you have seen them :).